src/index.js

```diff
@@ -17,8 +17,14 @@ const apps = require('./apps'),
       lockfile = require('./lockfile'),
       packageUtils = require('./package');
 
+const modeDefaults = {
+  appsToDeploy: APPS,
+  stageDeployment: true,
```

Comment:
Although the config interface is scenario driven, my goal here was to use multiple flags to help keep the underlying config more generic. I could also just do `mode.name === 'satellite'` or switch off one flag instead of many.